Dangerfile.js

```diff
@@ -1 +1,23 @@
-message("Hello, world.")
+// CHANGELOG check
+if (_.includes(git.modified_files, "CHANGELOG.md") === false) {
+  fail("No CHANGELOG added.")
+}
+
+const test_files = _.filter(git.modified_files, function(path){
```

Comment:
In enough time, I will create an eslint plugin for Danger, so that we'd get those messages inline